courses/intro-to-ada/intro_to_ada.rst

```diff
@@ -198,253 +207,427 @@ There are several note worthy things in the above program:
    create multiple lines of comments in Ada is by using :ada:`--` on each
    line. For example:
 
+.. admonition:: In other languages
+
+    Procedures ares similar to functions in C or C++ that return :c:`void`.
+    We'll see later how to declare functions in Ada.
+
 .. code-block:: ada
     :class: ada-nocheck
 
     --  We start a comment in this line...
     --  and we continue on the second line...
 
-Imperative language - If/Else
------------------------------
-
-Ada has an if statement. It is pretty unsurprising in form and function:
+Here is a minor variant of the "Hello, World" example:
 
 .. code-block:: ada
 
     with Ada.Text_IO; use Ada.Text_IO;
 
     procedure Greet is
-       I : Integer := 1;
     begin
-       loop
-          if I = 5 then
-             Put_Line ("Hello, World!");
-          end if;
-          I := I + 1;
-       end loop;
+       --  Print "Hello, World!" to the screen
+       Put_Line ("Hello, World!");
     end Greet;
 
-As for the while loop, the Boolean condition must be of strict type
-:ada:`Boolean`. Every relational operator in Ada returns a :ada:`Boolean`
-by default.
+This version utilizes an Ada feature known as a :ada:`use` clause, which has
+the the form ``use`` *package-name*. As illustrated by the call on
+:ada:`Put_Line`, the effect is that entities from the named package can be
+referenced directly, without the *package-name.* prefix.
+
+Imperative language - If/Then/Else
+----------------------------------
+
+This section describes Ada's ``if`` statement and introduces several other
```

Comment:
ada role missing here

Reply:
I'm seeing this role missing in multiple places. Is this distinction between "if" as a statement and as a keyword on purpose?